programs/libraries/rjson.scl

```diff
@@ -0,0 +1,109 @@
+//raw-json string to text
+if(0,    //if you are using an old version, change it to 1
+rjson(rj)->(
+barname='chyxrjson:rjson'+(rand(0.9)-'.');
+if(bossbar(barname)==false,return(rjson(rj)));
+command=run('bossbar set '+barname+' name '+rj);
+res=bossbar(barname,'name');
+bossbar(barname,'remove');
+if (command:1,res,throw('not valid r-json'))
+);
+
+
+,
+rjson(rj)->(
+r1=item_display_name(['apple',1,encode_nbt({'display'->{'Name'->rj}})]);
+r2=item_display_name(['stone',1,encode_nbt({'display'->{'Name'->rj}})]);
+if(r1==r2,r1,throw('not valid r-json'))
+);
+);
+
+//turns text-like values to a real text
+//will accept a [id, count, nbt] in the future
+__argument_rcv=(_(value)->(

```

Comment:
Couldn't you have just used a function here?

Reply:
I wrote it like this on purpose. Do not want to expose this function to the outside